lib/state/appActions.js

```diff
@@ -276,6 +276,7 @@ appActions.addOrRemoveTargetDevice = function(e) {
       return device === e.target.value;
     });
   }
+
```

Comment:
It doesn't matter very much, but please try not to check in meaningless changes. You should be reviewing the files you check in to look for stuff like this and revert them before committing.